### PR TITLE
[feature](statistics)Create partition statistics table.

### PR DESCRIPTION
**Pull request by Jibing-Li**
Automatically create partition_statistics for partition stats.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java [hunk 1]
```diff
@@ -158,8 +164,15 @@ public static void modifyTblReplicaCount(Database database, String tblName) {
...
-        Env.getCurrentEnv().getInternalCatalog().createTable(buildHistogramTblStmt());
+        Env.getCurrentEnv().getInternalCatalog().createTable(
+                buildStatisticsTblStmt(StatisticConstants.TABLE_STATISTIC_TBL_NAME,
+                    Lists.newArrayList("id", "catalog_id", "db_id", "tbl_id", "idx_id", "col_id", "part_id")));
+        Env.getCurrentEnv().getInternalCatalog().createTable(
+                buildStatisticsTblStmt(StatisticConstants.PARTITION_STATISTIC_TBL_NAME,
+                    Lists.newArrayList("catalog_id", "db_id", "tbl_id", "idx_id", "part_id", "col_id")));
+        // Env.getCurrentEnv().getInternalCatalog().createTable(
+        //         buildStatisticsTblStmt(StatisticConstants.HISTOGRAM_TBL_NAME,
+        //             Lists.newArrayList("id", "catalog_id", "db_id", "tbl_id", "idx_id", "col_id")));
```
[the remark this answers is not in the document]
Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java]: removed

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by Jibing-Li** [comment]
run cloud_p1

**Review by morrySnow — DISMISSED** [review]

**Comment by yiguolei** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run feut

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run cloud_p1

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run p0

**Comment by Jibing-Li** [comment]
run buildall

**Review by morningman — APPROVED** [review]

**Review by zddr — APPROVED** [review]

**Pull request merged**